pandas/compat/__init__.py

```diff
@@ -102,27 +102,7 @@ def is_platform_arm() -> bool:
     return platform.machine() in ("arm64", "aarch64")
 
 
-def import_lzma():
```

Comment:
Not sure if this is considered to be public API and would need a deprecation cycle?

Reply:
no its not